src/main/java/org/wildfly/security/authz/PermissionMapper.java

```diff
@@ -48,7 +48,7 @@
     PermissionCollection mapPermissions(Principal principal, Set<String> roles);
 
     /**
-     * A default implementation that does nothing but returns an empty and read-only {@link PermissionCollection}.
+     * A default implementation that does nothing but returns an empty and read-only {@link Policy#UNSUPPORTED_EMPTY_COLLECTION}.
```

Comment:
This change in specification is probably not warranted or desired - if this policy becomes problematic, we may have to replace it with a different empty one.